bcipy/parameters/parameters.json

```diff
@@ -628,32 +628,32 @@
     "recommended_values": "",
     "type": "str"
   },
-  "min_seq_len": {
+  "min_inq_len": {
     "value": "1",
     "section": "bci_config",
-    "readableName": "Minimum Sequence Length",
-    "helpTip": "The minimum number of sequences to present in spelling tasks",
+    "readableName": "Minimum inquiry Length",
```

Comment:
You might want to capitalize 'inquiry' here for consistency.

Reply:
Nice catch - I tried to find them all but I knew a few would slip through!